packages/devops/scripts/ci/validateBranchName.sh

```diff
@@ -4,7 +4,7 @@ DIR=$(dirname "$0")
 . ${DIR}/utils.sh
 
 INVALID_CHARS=('/' '\' '.' '&' '?')
-SUBDOMAIN_SUFFIXES=(admin aggregation api config export mobile psss psss-api report-api tonga-aggregation www)
+SUBDOMAIN_SUFFIXES=(admin aggregation api config export lesmis-api mobile psss psss-api report-api tonga-aggregation www)
```

Comment:
This reminds me of the requirement to add the subdomain to the EC2 tags. Is that in the docs already?

Reply:
yep it is